--- kernel/pf_ring.c.orig	2016-06-10 12:57:51.584308986 +0700
+++ kernel/pf_ring.c	2016-06-10 12:58:08.804504417 +0700
@@ -6945,11 +6945,7 @@
       if (unlikely(enable_debug))
         printk("[PF_RING] BPF filter (len = %u)\n", fprog.len);
 
-#if(LINUX_VERSION_CODE >= KERNEL_VERSION(4,4,0)) /* FIXX ubuntu only */
-      ret = __sk_attach_filter(&fprog, pfr->sk, sock_owned_by_user(pfr->sk));
-#else
       ret = sk_attach_filter(&fprog, pfr->sk);
-#endif
 
       if (ret == 0)
         pfr->bpfFilter = 1;
@@ -6959,11 +6955,7 @@
   case SO_DETACH_FILTER:
     if (unlikely(enable_debug))
       printk("[PF_RING] Removing BPF filter\n");
-#if(LINUX_VERSION_CODE >= KERNEL_VERSION(4,4,0)) /* FIXX ubuntu only */
-    ret = __sk_detach_filter(pfr->sk, sock_owned_by_user(pfr->sk));
-#else
     ret = sk_detach_filter(pfr->sk);
-#endif
     pfr->bpfFilter = 0;
     break;
 
